schema_utils.py

```diff
@@ -64,7 +64,8 @@
 EMAIL_REGEX = r'[\w\.\+\-]+\@[\w]+\.[a-z]{2,3}'
 
 
-def normalize_against_schema(obj, schema, apply_custom_validators=True):
+def normalize_against_schema(
```

Comment:
Please add schema_utils tests to verify that the global validators are called when provided / fail when expected & provided.

Reply:
Done.